kovri.i2p/_i18n/fr/contributing.md

```diff
@@ -78,76 +79,96 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
 - Maintainers SHALL have commit access to the repository.
 - Everyone, without distinction or discrimination, SHALL have an equal right to become a Contributor under the terms of this contract.
 
-### Licensing and Ownership
+- Le projet UTILISE le système de révision de contrôle fourni par GIT.
```

Comment:
I used [this translation of RFC 2119]( http://abcdrfc.free.fr/rfc-vf/rtf/rfc2119.rtf). Those are codified words and imho, we don't have a choice.